runner/scripts/3-runAssignments.ts

```diff
@@ -1,45 +1,134 @@
 import * as Path from 'path';
-import IAssignment from '@double-agent/collect-controller/interfaces/IAssignment';
-import Core from '@ulixee/hero-core';
-import Hero from '@ulixee/hero-fullstack';
-import RealUserAgents from '@double-agent/real-user-agents';
-import runAssignmentInHero from '../lib/runAssignmentInHero';
+import { HeroRunnerFactory } from '../lib/runAssignmentInHero';
+import { SecretAgentRunnerFactory } from '../lib/runAssignmentInSecretAgent';
+import { PuppeteerRunnerFactory } from '../lib/runAssignmentInPuppeteer';
 import forEachAssignment from '../lib/forEachAssignment';
+import { IRunnerFactory, IRunner } from '../interfaces/runner';
+import { program } from 'commander';
+import { exit } from 'process';
+
+// RunnerID groups together all supported runner implementations.
+enum RunnerID {
```

Comment:
Code convention for acronyms (even things like ID) is to name this as `RunnerId`.

Reply:
Totally agree. I had to be convinced of this one ;)